keycloak/themes/uid2-theme/login/messages/messages_en.properties

```diff
@@ -1,16 +1,18 @@
 # encoding: UTF-8
-doLogIn=Login
+doLogIn=Log In
 noAccount=Don’t have an account?
-doRegister=Create Account
-loginAccountTitle=Login
-registerTitle=Create an Account
-registerInfo=No worries, this isn’t set in stone. You can update these sections at anytime moving forward. 
+doRegister=Request Account
+loginAccountTitle=Log In
+registerTitle=Request Account
+registerInfo=To request an account, provide the following information.
 infoTitle=Your Information
 passwordTitle=Create a Password
 doRegisterButton=Continue
 errorPatternNoMatch=We’re sorry, but we only accept sign-ups from company email addresses. Please use your official company email to register and try again.
-forgotPasswordInfo=Enter your email address below and we will send a link to reset your password.
-doSendLink=Send Link
+forgotPasswordInfo=Enter your email address and we’ll send a link to reset your password.
+doSendLink=Request Password Reset
 emailForgotTitle=Forgot Password
 updatePasswordTitle=Reset Password
```

Comment:
We can not differentiate if this user goes to this page by invite or forgot password. i saw on figma it says 'Create password' for new users, should we change this for a generic one that works for both?